### PR TITLE
Fix #555 document rules for key assignment

### LINKED ISSUE #555: Document precise cases when assigning a consumer key K is allowed
# Problem

The rules for assigning a key are not completely trivial due to needing to make things secure. This needs to be documented.

# Closing criteria

Document clearly somewhere

# Problem details

None

# TODOs
- [x] Labels have been added for issue
- [x] Issue has been added to the ICS project

### PR DESCRIPTION
**Pull request by danwt**
# Description

Adds short description of rules about when a validator can and can't assign a given key.

## Linked issues

Closes: https://github.com/cosmos/interchain-security/issues/555

## Type of change

- [x] `Docs`: Adds documentation

### CODE REVIEW

File: docs/info_for_validators.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+##  Key Assignment
+
+The key assignment feature enables validators to use different consensus keys for each consumer chain. The feature allows validators to submit an AssignConsumerKey transaction to the provider chain. Pseudocode for type:
+
+```txt
+message MsgAssignConsumerKey {
+  // The chain id of the consumer chain to assign a consensus public key to
+  string chain_id;
+  // The validator address on the provider
```
Comment (smarshall-spitzbart) [inline docs/info_for_validators.md:13]: Operator addr or cons addr? See https://github.com/cosmos/interchain-security/pull/672#discussion_r1084325871.

We should consider renaming/documenting this field accordingly

**Review by smarshall-spitzbart — APPROVED** [review]
LGTM besides comment on address type

File: docs/info_for_validators.md (hunk 1 above)
Comment (smarshall-spitzbart) [reply docs/info_for_validators.md:13]: cc @mpoke

Comment (mpoke) [reply docs/info_for_validators.md:13]: Operator address. See https://github.com/cosmos/interchain-security/blob/b9d62995662a86c5b8eeb053c245ecfbd801458f/x/ccv/provider/keeper/msg_server.go#L44

**Review by sainoe — APPROVED** [review]
LGTM. It could be useful to explain here or in the README how to use the `assign-consensus-key` cmd from the CLI.

**Pull request merged**